packages/amplify-graphql-relational-transformer/src/graphql-has-many-transformer.ts

```diff
@@ -53,6 +53,7 @@ import {
   validateRelatedModelDirective,
 } from './utils';
 import { getGenerator } from './resolver/generator-factory';
+import { is } from 'immer/dist/internal';
```

Comment:
Is this an accidental import? (I had one of those earlier today, so if so, I sympathize. :) )

Reply:
Yes, this sneaked in my commit. Removed.